### PR TITLE
Merge my branch to master

### PR DESCRIPTION
**Pull request by BehnamBorhani**
add some custom node to stencil

### CODE REVIEW

File: src/app/main/main.component.scss [hunk 1]
```diff
@@ -1,6 +1,6 @@
 .wrapper {
     display: grid;
-    grid-template-columns: 180px 1fr;
+    grid-template-columns: 330px 1fr;
```
Comment (Pouya2781) [inline src/app/main/main.component.scss]: change to rem

Comment (BehnamBorhani) [reply src/app/main/main.component.scss]: fixed

**Review by Pouya2781 — CHANGES_REQUESTED** [review]
change

File: src/app/main/main.component.scss [hunk 2]
```diff
@@ -1,6 +1,6 @@
 .wrapper {
     display: grid;
-    grid-template-columns: 180px 1fr;
+    grid-template-columns: 33rem 1fr;
```
Comment (Pouya2781) [inline src/app/main/main.component.scss:3]: this is ok

**Review by Pouya2781 — CHANGES_REQUESTED** [review]
change it

**Review by Pouya2781 — APPROVED** [review]
ok

**Pull request merged**